sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/ItemsTable.scala

```diff
@@ -67,8 +67,6 @@ private class ItemsTable(val sparkSession: SparkSession,
   override def name(): String = tableName
 
   override def capabilities(): util.Set[TableCapability] = Set(
-    // ACCEPT_ANY_SCHEMA is needed because of this bug https://github.com/apache/spark/pull/30273
-    // It was fixed in Spark 3.1.0 but Databricks currently only supports 3.0.1
     TableCapability.ACCEPT_ANY_SCHEMA,
```

Comment:
Is this needed now? If we are using Spark 3.1.1?

Reply:
as per javadoc:

`ACCEPT_ANY_SCHEMA` Signals that the table accepts input of any schema in a write operation.

https://spark.apache.org/docs/3.1.1/api/java/org/apache/spark/sql/connector/catalog/TableCapability.html#ACCEPT_ANY_SCHEMA

As per its description it seems its purpose is something else and not relevant to the bug in spark 3.0, that's why I kept it.